### PR TITLE
add ability to rotate/flip images

### PR DESCRIPTION
**Pull request by Dillonmcewan**
https://www.pivotaltracker.com/story/show/131740579

Need to conform to methods called by super of Engine:

https://www.dropbox.com/s/4eqc2kc75ynh5y9/Screenshot%202016-10-24%2015.07.15.png?dl=0

### CODE REVIEW

File: opencv_engine/engine.py [hunk 1]
```diff
@@ -17,7 +17,6 @@
 
 from thumbor.engines import BaseEngine
 from pexif import JpegFile, ExifSegment
-import cv2
```
Comment (mikedeploysdrones) [inline opencv_engine/engine.py]: how are cv and cv2 imported?

Comment (Dillonmcewan) [inline opencv_engine/engine.py]: Imported above. This was a duplicate import

Comment (mikedeploysdrones) [inline opencv_engine/engine.py]: ah thx, I see now, had accordioned down but not up

**Comment by mikedeploysdrones** [comment]
LGTM

**Pull request merged**